pkg/statsd/latency_histogram.go

```diff
@@ -0,0 +1,74 @@
+package statsd
+
+import (
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/atlassian/gostatsd"
+)
+
+const (
+	HistogramThresholdsTagPrefix = "gsd_histogram:"
+	HistogramThresholdsSeparator = "_"
+)
+
+func LatencyHistogram(timer gostatsd.Timer) map[gostatsd.HistogramThreshold]int {
```

Comment:
Doesn't need to be exported.

Reply:
I lack golang knowledge, how can I use it outside in `aggregator` then?